### DISCUSSION TITLE
Coordinate system for custom positioning system

### DISCUSSION THREAD

**Discussion by ar-ramchandra** [bitcraze/discussions#1716] [Q&A]
If I use my own motion capture system to calculate the 3d position of the crazyflie, and send it over to the drone via the api, what would it expect in terms of position ? What units must X,y,z be (I think it's meters but just want to make sure)? 
If the drone is on the floor will its position be 0,0,0 or some X,y,0 (the last value is 0 because it's altitude is 0, but X and y are not because it's somewhere within my definition of a coordinate system) based on my definition of a coordinate system?

Can X,y,z be floats ? If so how many points of precision is accepted ? 

Any advice is deeply appreciated! 
Thanks in advance !

**Comment by ToveRumar** [comment]
Hi @ar-ramchandra!

The motion capture lib is not maintained by us, but I think you will find answers to your questions by checking out the API https://github.com/IMRCLab/libmotioncapture

My guess is that what unit and precision that can be used depends on the setup of your mocap system. 

I hope this could help you find answers to your question.
